src/adapters/sekindoUM.js

```diff
@@ -0,0 +1,110 @@
+import { getBidRequest } from '../utils.js';
+var CONSTANTS = require('../constants.json');
+var utils = require('../utils.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+
+var sekindoUMAdapter;
+sekindoUMAdapter = function sekindoUMAdapter() {
+
+  function _callBids(params) {
+    var bids = params.bids;
+    var bidsCount = bids.length;
+
+    var pubUrl = null;
+    if (parent !== window)
+      pubUrl = document.referrer;
+    else
+      pubUrl = window.location.href;
+
+    for (var i = 0; i < bidsCount; i++) {
+      var bidReqeust = bids[i];
+      var callbackId = bidReqeust.bidId;
+      _requestBids(bidReqeust, callbackId, pubUrl);
+      //store a reference to the bidRequest from the callback id
+      //bidmanager.pbCallbackMap[callbackId] = bidReqeust;
+    }
+  }
+
+  $$PREBID_GLOBAL$$.sekindoCB = function(callbackId, response) {
+    var bidObj = getBidRequest(callbackId);
+    if (typeof (response) !== 'undefined' && typeof (response.cpm) !== 'undefined') {
+      var bid = [];
+      if (bidObj) {
+        var bidCode = bidObj.bidder;
+        var placementCode = bidObj.placementCode;
+
+        if (response.cpm !== undefined && response.cpm > 0) {
+
+          bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD);
+          bid.callback_uid = callbackId;
+          bid.bidderCode = bidCode;
+          bid.creative_id = response.adId;
+          bid.cpm = parseFloat(response.cpm);
+          bid.ad = response.ad;
+          bid.width = response.width;
+          bid.height = response.height;
+
+          bidmanager.addBidResponse(placementCode, bid);
+        }
+
+        else {
+          bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID);
+          bid.callback_uid = callbackId;
+          bid.bidderCode = bidCode;
+          bidmanager.addBidResponse(placementCode, bid);
+        }
+      }
+    }
+
+    else {
+      if (bidObj) {
+        utils.logMessage('No prebid response for placement '+bidObj.placementCode);
+      }
+
+      else {
+        utils.logMessage('sekindoUM callback general error');
+      }
+    }
+  };
+
+  function _requestBids(bid, callbackId, pubUrl) {
+    //determine tag params
+    var spaceId = utils.getBidIdParamater('spaceId', bid.params);
+    var subId = utils.getBidIdParamater('subId', bid.params);
+    var bidfloor = utils.getBidIdParamater('bidfloor', bid.params);
```

Comment:
This utils method was changed from `getBidIdParamater` to `getBidIdParameter`, make sure to update this and the previous two lines to match so that this will work again